src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/FilterDelegationBase.cs

```diff
@@ -184,6 +184,19 @@ protected bool IsValidDelegatableFilterPredicateNode(TexlNode dsNode, TexlBindin
             return true;
         }
 
+        public override void CheckSemantics(TexlBinding binding, TexlNode[] args, DType[] argTypes, IErrorContainer errors)
```

Comment:
>CheckSemantics [](http://example.com/codeflow?start=29&length=14)

While I think this is the right thing to do, this is technically a breaking change for Power Apps, so we should only enforce this check with PowerFxV1CompatibilityRules.